### PR TITLE
update docs/devel flaky-tests to v1beta3

### PR DESCRIPTION
**Pull request by caesarxuchao**
update docs/devel flaky-tests to v1beta3
@nikhiljindal

### CODE REVIEW

File: docs/devel/flaky-tests.md [hunk 1]
```diff
@@ -11,32 +11,29 @@ There is a testing image ```brendanburns/flake``` up on the docker hub.  We will
...
-            value: https://github.com/GoogleCloudPlatform/kubernetes
-      restartpolicy: {}
-    labels:
-      name: flake
-labels:
-  name: flake
+  template:
+    metadata:
+      labels:
+        name: flake
```
Comment (nikhiljindal) [inline docs/devel/flaky-tests.md]: too many "name: flake" here :)
@bgrant0607 mentioned that we now populate the RC selector and labels from the pod template's labels by default.
We should take advantage of that and can skip the RC selector and labels.

Comment (bgrant0607) [inline docs/devel/flaky-tests.md]: Yes, please try it.

File: docs/devel/flaky-tests.md [hunk 2]
```diff
@@ -11,32 +11,29 @@ There is a testing image ```brendanburns/flake``` up on the docker hub.  We will
...
+    spec:
+      containers:
+      - env:
+        - name: TEST_PACKAGE
+          value: pkg/tools
+        - name: REPO_SPEC
+          value: https://github.com/GoogleCloudPlatform/kubernetes
+        image: brendanburns/flake
```
Comment (bgrant0607) [inline docs/devel/flaky-tests.md]: I assume the conversion tool put image and name at the end -- alphabetical order. However, it's more readable to put name, image, then the rest.

File: docs/devel/flaky-tests.md (hunk 1 above)
Comment (caesarxuchao) [inline docs/devel/flaky-tests.md]: It's good to know. Thanks.

**Comment by caesarxuchao** [comment]
@nikhiljindal @bgrant0607 Comments addressed. Please take another look. Thanks.

**Comment by nikhiljindal** [comment]
LGTM, thanks!

**Comment by caesarxuchao** [comment]
ref #8087

**Pull request merged**